### PR TITLE
feat(thumbnail) Video thumbnails redesign and refactor

### PR DESCRIPTION
**Pull request by robertpin**
Update video thumbnail design
Update design of indicators
In filmstrip view move Screen Sharing indicator to the top
Removed dominant speaker indicator
Use ContextMenu component for the connection stats popover
Combine Remove video menu and Meeting participant context menu into one component
Moved some styles from SCSS to JSS
Fix mobile avatars too big
Fix mobile horizontal scroll

### CODE REVIEW

**Comment by robertpin** [comment]
Tile view
vertical filmstrip
horizontal filmstrip

**Comment by vp8x8** [comment]
Left a few comments. 

!NOTE: Before the PR is merged we have to make sure large meetings still will work well (there are no unnecessary re-renders which make the app freeze.)

**Comment by robertpin** [comment]
Jenkins test this please

**Comment by robertpin** [comment]
Jenkins test this please

File: react/features/video-menu/components/web/AskToUnmuteButton.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+    return (
+        <ContextMenuItem
+            accessibilityLabel = { displayUnmuteText
+                ? t('participantsPane.actions.askUnmute')
+                : t('participantsPane.actions.allowVideo')
+            }
+            icon = { IconMicrophoneEmpty }
+            onClick = { _onClick }
+            text = { displayUnmuteText
```
Comment (vp8x8) [inline react/features/video-menu/components/web/AskToUnmuteButton.js:46]: I see this ternary used twice.

File: react/features/filmstrip/components/web/VideoMenuTriggerButton.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+const VideoMenuTriggerButton = ({
+    hidePopover,
+    local,
+    participantId,
+    popoverVisible,
+    showPopover,
+    visible
+}: Props) => {
+    const localMenu = (
```
[the remark this answers is not in the document]
Comment (robertpin) [reply react/features/filmstrip/components/web/VideoMenuTriggerButton.js:48]: If I don't  eslint gets in a weird state where it has two errors: fixing one results in the other. So it's either this or disable eslint line

**Review by vp8x8 — APPROVED** [review]

**Review by vp8x8 — DISMISSED** [review]

**Comment by robertpin** [comment]
Jenkins test this please

**Comment by robertpin** [comment]
Jenkins test this please

**Comment by robertpin** [comment]
Jenkins test this please

**Comment by robertpin** [comment]
Jenkins test this please

**Comment by robertpin** [comment]
Jenkins test this please

**Comment by robertpin** [comment]
Jenkins test this please

**Comment by robertpin** [comment]
Jenkins test this please

**Comment by robertpin** [comment]
Jenkins test this please

**Review by horymury — APPROVED** [review]

**Pull request merged**